src/templates/PoolV2/index.tsx

```diff
@@ -111,6 +109,7 @@ const Pool = () => {
 
   const router = useRouter()
   const { data: pool } = usePoolData({ id: router.query.address as string })
+  const { data: tokenSwap } = useTokenSwap({ chainId: pool?.chain_id ?? 137 })
```

Comment:
Does "137" make sense for everyone reading this code? if yes, you can keep it, if it's something obscure that you should have lots of context to understand, consider giving it a variable.

```suggestion
  const { data: tokenSwap } = useTokenSwap({ chainId: pool?.chain_id ?? 137 })
```